ingestion_tools/scripts/data_validation/tests/annotation/test_segmentationmask_annotation.py

```diff
@@ -0,0 +1,308 @@
+from typing import Dict
+
+import numpy as np
+import pytest
+from mrcfile import utils
+from mrcfile.mrcinterpreter import MrcInterpreter
+
+
+# By setting this scope to session, scope="session" fixtures will be reinitialized for each run + voxel_spacing combination
+@pytest.mark.annotation
+@pytest.mark.parametrize("run_name, voxel_spacing", pytest.run_spacing_combinations, scope="session")
+class TestSegmentationMaskHeader:
+    MAP_ID = b"MAP "
+    VOLUME_SPACEGROUP = 1
+    # Tolerance for voxel spacing assertions
+    SPACING_TOLERANCE = 0.01
+    ZATTRS_AXIS_ORDER = ["z", "y", "x"]
+
+    """Validate the mrc file header for a volume annotation."""
+
+    ### BEGIN MRC Self-consistency tests ###
+    def header_helper(self, seg_mask_annotation_mrc_headers: Dict[str, MrcInterpreter], check_func: callable, **kwargs):
+        """Helper function to check the header of the mrc file."""
+        for mrc_filename, interpreter in seg_mask_annotation_mrc_headers.items():
+            print(f"MRC File: {mrc_filename}")
+            check_func(interpreter.header, interpreter, **kwargs)
+
```

Comment:
A helper function that reduces repetitiveness of code - takes a validator function to call on the header data. Mrc header validation functions below just define their own validator function and pass it in here